bokehjs/src/coffee/client.coffee

```diff
@@ -134,6 +134,8 @@ class ClientConnection
 
     try
       versioned_url = "#{@url}?bokeh-protocol-version=1.0&bokeh-session-id=#{@id}"
+      if @args_string?.length > 0
+        versioned_url += "&#{@args_string}"
```

Comment:
does this end up putting bokeh-protocol-version and bokeh-session-id in there twice or not? I'm not sure whether they could already be in the url